api/base/serializers.py

```diff
@@ -280,6 +299,54 @@ def to_representation(self, data):
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+    # Overrides ListSerializer which doesn't support multiple update by default
+    def update(self, instance, validated_data):
```

Comment:
I thought I did have unit tests for this.  All bulk update operations are using this - bulk updates of nodes and bulk updates of contributors.